.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
 
       - name: Run tests
-        run: go test ./bucket-blocker/...
+        run: go list -f '{{.Dir}}/...' -m | xargs go test
```

Comment:
This tests every module. Go doesn't have great support for running the same command across multiple workspaces as far as i can see